### DISCUSSION TITLE
弹幕越来越少了，看不到满屏弹幕少了B站基础乐趣了

### DISCUSSION THREAD

**Discussion by xiaoyanxyxy** [ywmoyue/biliuwp-lite#215] [General]
现在哔哩哔哩lite的弹幕越来越少了
只能显示三分之一多，没有以前满屏弹幕了
同一个屏幕几千或万的视频
手机、网页和电脑桌面客户端看，可以显示很多弹幕 
在lite中，就稀稀拉拉的，很多出不来
没有以前的密集程度了
是什么原因呢？

**Comment by ywmoyue** [comment]
b站换接口了

**Reply by xiaoyanxyxy** [reply]
老哥没有办法嘛，现在都只能看网页版弹幕了

**Comment by xiaoyanxyxy** [comment]
好久没过瘾弹幕了 何时能修复呢 一直都回归网页版了

**Reply by ywmoyue** [reply]
下个版本

**Comment by ywmoyue** [comment]
4.6.14 已修复
